engine/runtime-integration-tests/src/test/java/org/enso/compiler/ErrorCompilerTest.java

```diff
@@ -52,12 +52,7 @@ public void unaryMinus() throws Exception {
     main = Date.new day=-
     """);
 
-    assertSingleSyntaxError(
-        ir,
-        new Syntax.UnsupportedSyntax("Strange unary -"),
-        "Syntax is not supported yet: Strange unary -",
-        51,
-        52);
+    assertSingleSyntaxError(ir, Syntax.UnrecognizedToken$.MODULE$, "Unrecognized token", 51, 52);
```

Comment:
The error in this case has changed to a multiple-operator error. I think this makes more sense than the previous error type.